.vscode/settings.json

```
@@ -44,10 +44,18 @@
   },
   "jestrunner.debugOptions": {
     // Uncomment to debug integration tests
-    // "attachSimplePort": 9223,
+    "attachSimplePort": 9223,
```

This should remain commented.
```suggestion
    // "attachSimplePort": 9223,
```